beta/azure-container-apps/ReadMe.md

```diff
@@ -60,95 +55,129 @@ Deploying 1Password SCIM Bridge on Azure Container Apps comes with a few benefit
+   az containerapp env create --name $ConAppEnv --resource-group $ResourceGroup --location $Location --enable-workload-profiles false
    ```
 
-5. Upload your `scimsession` file to the Cloud Shell:
+7. Upload your `scimsession` file to the Cloud Shell:
    - Click the “Upload/Download files” button and choose Upload.
    - Find the `scimsession` file that you saved to your computer and choose it.
    - Make a note of the upload destination, then click Complete.
 
-6. Create the base Container App and Secret for your Base64-encoded `scimsession` credentials, _(using the bash or PowerShell syntax for the commands)_:
+8. Deploy the Container App Secret for the SCIM bridge. 
+
+    > **Note:** If you prefer to deploy the containers using the Azure Container App Commands, go to this section: [Azure ContainerApps Commands Deployment steps](#azure-container-apps-commands-deployment-steps), othersise follow the template file steps below. 
+    >
+
+    Create your Container App Secret:
     - Using bash
+        ```bash
+        az containerapp create --resource-group $ResourceGroup --environment $ConAppEnv --name $ConAppName  --secrets scimsession="$(cat /home/$USER/scimsession | base64)"
+        ```
+    - Using PowerShell 
+        ```pwsh
+        az containerapp create --resource-group $ResourceGroup --environment $ConAppEnv --name $ConAppName  --secrets scimsession="$([Convert]::ToBase64String([IO.File]::ReadAllBytes((Join-Path /home/$Env:USER/ 'scimsession'))) )"
+        ```
+
+9. Deploy your SCIM bridge containers based off the template file:
+
+    Upload the [aca-op-scim-bridge.yaml](aca-op-scim-bridge.yaml) file to the Cloud Shell:
+    - In a separate browser window, download the [aca-op-scim-bridge.yaml](aca-op-scim-bridge.yaml) (selecting the download icon from the top right) from our GitHub repository. 
+    - Click the “Upload/Download files” button and choose Upload.
+    - Find the `aca.op-scim-bridge.yaml` file that you saved to your computer and choose it.
+
+    Run the following command to deploy the Container App from the template file:
     ```bash
-    az containerapp create -n $ContainerAppName -g $ResourceGroup \
-        --container-name op-scim-bridge \
-        --image docker.io/1password/scim:v2.8.4 \
-        --environment $ContainerAppEnvironment \
-        --ingress external --target-port 3002 \
-        --cpu 0.25 --memory 0.5Gi \
-        --min-replicas 1 --max-replicas 1 \
-        --secrets scimsession="$(cat /home/$USER/scimsession | base64)" \
-        --env-vars OP_REDIS_URL="redis://localhost:6379" OP_SESSION=secretref:scimsession
+    az containerapp update --resource-group $ResourceGroup --name $ConAppName --yaml aca-op-scim-bridge.yaml --query properties.configuration.ingress.fqdn
+    ```
+
+10. Open the domain name listed in a separate browser tab to test your connection. You can sign in to your SCIM bridge using your bearer token.
```

Comment:
I've ran into this enough times, can we specify "https://foobar" here? I've had to tell folks "go ahead and put https:// ahead of it"